fe/fe-core/src/main/java/com/starrocks/fs/hdfs/HdfsFsManager.java

```diff
@@ -1238,7 +1238,10 @@ public void deletePath(String path, Map<String, String> loadProperties) throws U
     public void renamePath(String srcPath, String destPath, Map<String, String> loadProperties) throws UserException {
         WildcardURI srcPathUri = new WildcardURI(srcPath);
         WildcardURI destPathUri = new WildcardURI(destPath);
-        if (!srcPathUri.getAuthority().trim().equals(destPathUri.getAuthority().trim())) {
+        if  ((srcPathUri.getAuthority() == null && destPathUri.getAuthority() != null) ||
+                (srcPathUri.getAuthority() != null && destPathUri.getAuthority() == null) ||
+                    (srcPathUri.getAuthority() != null && destPathUri.getAuthority() != null && 
+                        (!srcPathUri.getAuthority().trim().equals(destPathUri.getAuthority().trim())))) {
             throw new UserException(
```

Comment:
The exception is not clear after change the comparison in the if branch.

Reply:
update the exception info